src/statd/statd.c

```diff
@@ -195,7 +206,38 @@ static int sr_ifaces_cb(sr_session_ctx_t *session, uint32_t, const char *path,
 		return SR_ERR_INTERNAL;
 	}
 
-	err = ly_add_yanger_data(ctx, parent, sub->ifname);
+	err = ly_add_yanger_data(ctx, parent, "ietf-interfaces", sub->name);
+	if (err)
+		ERROR("Error adding yanger data");
+
+	sr_release_context(con);
+
+	return err;
+}
+
+static int sr_routes_cb(sr_session_ctx_t *session, uint32_t, const char *path,
+                        const char *, const char *, uint32_t,
+                        struct lyd_node **parent, __attribute__((unused)) void *priv)
+{
+	const struct ly_ctx *ctx;
+	sr_conn_ctx_t *con;
+	sr_error_t err;
+
+	ERROR("Incoming query for xpath: %s", path);
```

Comment:
Accidental debug left? If you really want this line I think there should be some other macro with a more suitable name